tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+from bs4 import BeautifulSoup
+from pytest import fixture
+from selenium import webdriver
+
+from data import generate_form_data
+
+
+@fixture
+def set_up_browser():
```

Comment:
Здесь подойдет более обобщенное название.
```suggestion
def browser():
```